### ISSUE TITLE
how to change the trigger type of highlightplugin? #question 

### ISSUE THREAD

**Issue by napretep** [react-pdf-viewer/react-pdf-viewer#969]
```typescript
const highlightPluginInstance = highlightPlugin({
    renderHighlights,
    trigger: Trigger.None,
});
```
I want to both use select area and select text to trigger the highlightplugin, but the instance only pass the trigger type once.
where can I to change the trigger type when I want to switch?

**Comment by phuoc-ng** [comment]
It's not supported at the moment. I need to think of supporting this in the future version.

**Comment by Huynt-dev** [comment]
I have a similar problem about re-rendering the canvas when it was updated

**Comment by phuocng** [comment]
@napretep 
This feature request is implemented in the latest code, and will be available in the next v3.8.0.

## API

```ts
const highlightPluginInstance = highlightPlugin();
const { switchTrigger } = highlightPluginInstance;

// Switch to None
switchTrigger(Trigger.None);

// Switch to Selection mode
switchTrigger(Trigger.TextSelection);
```

## Demo 

https://user-images.githubusercontent.com/57786711/197381267-d4584725-47b3-4e19-a631-db3066046c56.mp4

**Comment by phuocng** [comment]
@napretep [v3.8.0](https://react-pdf-viewer.dev/updates/) is released. Please upgrade.
